examples/with-framer-motion/README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to Remix + framer-motion!
```

Comment:
Could you update this README to conform to the template for examples? (You can copy the template directory when creating examples to help ensure the example is simple and consistent).